src/Tools/BuildValidator/BuildConstructor.cs

```diff
@@ -88,7 +88,10 @@ void logResolvedSources()
                 {
                     var sourceFileInfo = resolvedSource.SourceFileInfo;
                     var hash = BitConverter.ToString(sourceFileInfo.Hash).Replace("-", "");
-                    _logger.LogInformation($@"""{resolvedSource.DisplayPath}"" - {sourceFileInfo.HashAlgorithm} - {hash}");
+                    var embeddedCompressedHash = sourceFileInfo.EmbeddedCompressedHash is { } compressedHash
```

Comment:
This was needed to verify that compression of embedded sources was producing equivalent results in the rebuild.